tests/templates/pr-create.md.ejs.js

```diff
@@ -127,28 +127,30 @@ describe("pr-create.md.ejs", function() {
         assert.ok(result.indexOf("require an issue") === -1);
     });
 
-    it("should not mention missing issue or length check when there's one fix commit", function() {
-        var result = ejs.render(TEMPLATE_TEXT, {
-            payload: {
-                sender: {
-                    login: "nzakas"
+    ["Breaking", "Build", "Chore", "Docs", "Fix", "New", "Update", "Upgrade"].forEach(function(type) {
```

Comment:
This is new territory for me in the codebase, but is this supposed to only be for "fix" PR's? If so, still love the idea of testing all the labels! Could maybe put that here: https://github.com/eslint/eslint/pull/5929/files#diff-5f03052725c45afbab9bb19b23b9494dR29